<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
	
	background-color: rgb(19, 99, 119);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_top">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#frame_top{
	
	background-color: rgb(252, 236, 12);

}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>740</x>
            <y>-10</y>
            <width>82</width>
            <height>44</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton{
background-color: rgba(255, 255, 255,0);
boder:none;
}
QPushButton:hover{
padding-bottom:5px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ico.qrc">
               <normaloff>:/ico/ico/minimize.png</normaloff>:/ico/ico/minimize.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true">QPushButton{
background-color: rgba(255, 255, 255,0);
boder:none;
}
QPushButton:hover{
padding-bottom:5px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ico.qrc">
               <normaloff>:/ico/ico/close.png</normaloff>:/ico/ico/close.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_content">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>95</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_left">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#frame_left{
	
	background-color: rgb(8, 74, 90);

}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QFrame" name="frame_operate">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>110</y>
               <width>171</width>
               <height>251</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_transfer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>851tegakizatsu</family>
                  <pointsize>15</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">
QPushButton{	border-radius:30px;
color: rgb(248, 237, 83);
}
QPushButton:pressed{	
image: url(:/ico/ico/cutbutton.png);
}</string>
                </property>
                <property name="text">
                 <string>文件传输</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_log">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>851tegakizatsu</family>
                  <pointsize>15</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">
QPushButton{	border-radius:30px;
color: rgb(248, 237, 83);
}
QPushButton:pressed{	
image: url(:/ico/ico/cutbutton.png);
}</string>
                </property>
                <property name="text">
                 <string>日志查看</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_info">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>851tegakizatsu</family>
                  <pointsize>15</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">
QPushButton{	border-radius:30px;
color: rgb(248, 237, 83);
}
QPushButton:pressed{	
image: url(:/ico/ico/cutbutton.png);
}</string>
                </property>
                <property name="text">
                 <string>软件信息</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="label_back">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>91</width>
               <height>61</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/ico/ico/redpoint2.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_user">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>0</y>
               <width>81</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Comic Sans MS</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(248, 237, 83);</string>
             </property>
             <property name="text">
              <string>Superyu</string>
             </property>
            </widget>
            <widget class="QFrame" name="frame_connect">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>470</y>
               <width>121</width>
               <height>51</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_connect_light">
                <property name="styleSheet">
                 <string notr="true">image: url(:/ico/ico/greenpoint.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_connect">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(248, 237, 83);</string>
                </property>
                <property name="text">
                 <string>Connected</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QFrame" name="frame_net">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>400</y>
               <width>111</width>
               <height>71</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_ip">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(248, 237, 83);</string>
                </property>
                <property name="text">
                 <string>IP:10.191.1.1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_port">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(248, 237, 83);</string>
                </property>
                <property name="text">
                 <string>Port:6666</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="label_user_identity">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>40</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>851tegakizatsu</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
color: rgb(248, 237, 83);</string>
             </property>
             <property name="text">
              <string>管理员</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_10">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>20</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/ico/ico/user1.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_right">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>8</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_right_top">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>2</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>90</x>
                  <y>0</y>
                  <width>491</width>
                  <height>81</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <pointsize>35</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(248, 237, 83);</string>
                </property>
                <property name="text">
                 <string>Secure File Transfer</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_right_bottom">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>8</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QStackedWidget" name="stackedWidget">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(0, 0, 0,0);</string>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="page_transfer">
                   <widget class="QListView" name="listView">
                    <property name="geometry">
                     <rect>
                      <x>30</x>
                      <y>20</y>
                      <width>551</width>
                      <height>251</height>
                     </rect>
                    </property>
                   </widget>
                   <widget class="QPushButton" name="pushButton_upload">
                    <property name="geometry">
                     <rect>
                      <x>110</x>
                      <y>290</y>
                      <width>111</width>
                      <height>41</height>
                     </rect>
                    </property>
                    <property name="font">
                     <font>
                      <family>851tegakizatsu</family>
                      <pointsize>15</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">
QPushButton{	
background-color: rgb(207, 44, 24);
color: rgb(255, 255, 255);
}
QPushButton:pressed{	
	padding-left:5px;
	padding-top:5px;
}</string>
                    </property>
                    <property name="text">
                     <string>上传</string>
                    </property>
                   </widget>
                   <widget class="QPushButton" name="pushButton_download">
                    <property name="geometry">
                     <rect>
                      <x>370</x>
                      <y>290</y>
                      <width>111</width>
                      <height>41</height>
                     </rect>
                    </property>
                    <property name="font">
                     <font>
                      <family>851tegakizatsu</family>
                      <pointsize>15</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">
QPushButton{	
background-color: rgb(207, 44, 24);
color: rgb(255, 255, 255);
}
QPushButton:pressed{	
	padding-left:5px;
	padding-top:5px;
}</string>
                    </property>
                    <property name="text">
                     <string>下载</string>
                    </property>
                   </widget>
                  </widget>
                  <widget class="QWidget" name="page_log"/>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
         <zorder>frame_right</zorder>
         <zorder>frame_left</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>765</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
